tests/test_browser.py

```diff
@@ -4265,6 +4265,24 @@ def test_webgl_draw_base_vertex_base_instance(self):
                          '-DEXPLICIT_SWAP=1',
                          '-DWEBGL_CONTEXT_VERSION=2'])
 
+  @requires_graphics_hardware
+  def test_webgl_sample_query(self):
+    cmd = ['-s', 'MAX_WEBGL_VERSION=2', '-lGL']
+    self.btest('webgl_sample_query.cpp', expected='0', args=cmd)
+
+  @requires_graphics_hardware
+  def test_webgl_timer_query(self):
+    for args in [
+        # EXT query entrypoints on WebGL 1.0
+        ['-s', 'MAX_WEBGL_VERSION'],
```

Comment:
I'm not sure, is this the same as `MAX_WEBGL_VERSION=1`? Could we just explicitly specify `=1`?

Reply:
Honestly I have no idea. Copypasted it from another occurence in this file which led me to believe it indeed is an implicit `=1` :)

Reply:
Yes, emcc allows `-s X` instead of `-s X=1`.